fed/mosaic-sumo/src/test/java/org/eclipse/mosaic/fed/sumo/bridge/TraciTest.java

```diff
@@ -413,12 +414,12 @@ public void testEmissionsAndFuelConsumptionCalculation() throws Exception {
 
         // ASSERT (for emission class HBEFA3/PC_G_EU4: http://sumo.dlr.de/wiki/Models/Emissions/HBEFA3-based)
         double per1kmFactor = 1000 / vehData.getDistanceDriven();
-        assertEquals(54.5, vehData.getVehicleConsumptions().getAllConsumptions().getFuel() * per1kmFactor, 1d);
+        assertEquals(69, vehData.getVehicleConsumptions().getAllConsumptions().getFuel() * per1kmFactor, 1d);
         assertEquals(490, vehData.getVehicleEmissions().getAllEmissions().getCo() * per1kmFactor, 2d);
-        assertEquals(126800, vehData.getVehicleEmissions().getAllEmissions().getCo2() * per1kmFactor, 100d);
+        assertEquals(161000, vehData.getVehicleEmissions().getAllEmissions().getCo2() * per1kmFactor, 100d);
         assertEquals(4.7, vehData.getVehicleEmissions().getAllEmissions().getHc() * per1kmFactor, 1d);
-        assertEquals(43.9, vehData.getVehicleEmissions().getAllEmissions().getNox() * per1kmFactor, 1d);
-        assertEquals(1.23, vehData.getVehicleEmissions().getAllEmissions().getPmx() * per1kmFactor, 0.1d);
+        assertEquals(52, vehData.getVehicleEmissions().getAllEmissions().getNox() * per1kmFactor, 1d);
+        assertEquals(1.35, vehData.getVehicleEmissions().getAllEmissions().getPmx() * per1kmFactor, 0.1d);
```

Comment:
SUMO previously returned 0 for emissions or consumptions whenever a vehicle reduced its speed. This is now handled slightly different, thus resulting in greater emissons/consumptions.